app/controllers/recipe_foods_controller.rb

```diff
@@ -0,0 +1,32 @@
+class RecipeFoodsController < ApplicationController
+  def new
+    @recipe_food = RecipeFood.new
+  end
+
+  def create
```

Comment:
- [ ] Good job so far but I noticed an issue with **the generation of the shopping list**. As the previous reviewer mentioned make sure that that lists are generated based on the ingredients added to a particular recipe. Also, make sure that all information about estimated quantity and price is displayed according to the project requirement. 😄 

#### Wireframe
![general_shopping_list (1)](https://github.com/brainskev/Recipe-app/assets/6615394/61f61cda-ded1-4661-9929-a0e5f954f407)

#### Your app

Reply:
@codecaiine I don't think that is what the project requirements meant. see it below the requirement said for all recipes not particular recipe as you have stated:
General shopping list view (only for a group that has less than 3 members):
    Should show the list of food that is missing for all recipes of the logged-in user (compare the list of food for all recipes with the general food list of that user).
    Should count the total food items and total price of the missing food.